WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1614,13 +1614,20 @@ extension WooAnalyticsEvent {
         }
 
         enum Name: String {
-            case todayStats = "today-stats"
+            case todayStats = "stats-today"
+            case appLink = "app-link"
         }
 
         /// Event when a widget is tapped and opens the app.
         ///
-        static func widgetTapped(name: Name) -> WooAnalyticsEvent {
-            WooAnalyticsEvent(statName: .widgetTapped, properties: [Key.name.rawValue: name.rawValue])
+        static func widgetTapped(name: Name, family: String? = nil) -> WooAnalyticsEvent {
+            let properties: [String: WooAnalyticsEventPropertyType]
+            if let family {
+                properties = [Key.name.rawValue: "\(name.rawValue)-\(family)"]
+            } else {
+                properties = [Key.name.rawValue: name.rawValue]
+            }
+            return WooAnalyticsEvent(statName: .widgetTapped, properties: properties)
```

Comment:
Do you think it would be better if the family is a property of the event? That way we can continue to be on par with android events?

Reply:
But then widgets names will be different between application_opened and widget_tapped?

Reply:
`application_opened` has string "widgets": "today-stats,today-stats" now. Where do we add `family`?